client/src/pages/create/SignInToCreatePage.tsx

```diff
@@ -38,7 +41,7 @@ export default function SignInToCreatePage() {
         <div className="flex flex-col gap-4">
           <Button
             onClick={() => openModal(LoginModalView.SIGNUP)}
-            className="self-center w-[150px]"
+            className="self-center w-36"
           >
```

Comment:
random code review changes, but main change related to pr is passing loadingAppUser